spec/beaker-answers/versions/version20162_spec.rb

```diff
@@ -176,192 +171,3 @@
       end
     end
 end
-
-describe BeakerAnswers::Version20162 do
-  let( :ver )         { '2016.2.0' }
-  let( :options )     { StringifyHash.new }
-  let( :basic_hosts ) { make_hosts( {'pe_ver' => ver } ) }
-  let( :hosts ) { basic_hosts[0]['roles'] = ['master', 'agent']
-                  basic_hosts[1]['roles'] = ['dashboard', 'agent']
-                  basic_hosts[2]['roles'] = ['database', 'agent']
-                  basic_hosts }
-  let( :answers )     { BeakerAnswers::Answers.create(ver, hosts, options) }
-  let( :answer_hash ) { answers.answers }
-
-  it 'adds orchestrator database answers to console' do
-    expect( answer_hash['vm2'][:q_orchestrator_database_name] ).to be === 'pe-orchestrator'
-    expect( answer_hash['vm2'][:q_orchestrator_database_user] ).to be === 'Orc3Str8R'
-  end
-
-  it 'generates valid answers if #answer_string is called' do
-    expect( answers.answer_string(basic_hosts[2]) ).to match(/q_orchestrator_database_name=pe-orchestrator/)
-  end
-
-  context 'when generating a hiera config' do
```

Comment:
I think we just want to remove the bash case from the describe block, and keep the describe and the hiera config case, otherwise this spec isn't testing anything.  Because the block up above is just a shared context referenced in include_examples below.